code/controllers/subsystems/skybox.dm

```diff
@@ -68,11 +68,6 @@ SUBSYSTEM_DEF(skybox)
 /datum/controller/subsystem/skybox/proc/get_skybox(z)
 	if(!skybox_cache["[z]"])
 		skybox_cache["[z]"] = generate_skybox(z)
-		if(global.using_map.use_overmap)
```

Comment:
Ah well. I guess we can't use this optimization because it ignores the possibility of using_map itself having specified different backgrounds.